core/domain/html_cleaner.py

```diff
@@ -400,7 +455,36 @@ def convert_to_text_angular(html_data):
     # html strings they are stored as <br>. Since both of these
     # should match and <br> and <br/> have same working,
     # so the tag has to be replaced in this way.
-    return str(soup).replace('<br/>', '<br>')
+    return unicode(soup).replace('<br/>', '<br>')
+
+
+def convert_tag_contents_to_text_angular(html_data):
+    """This functions converts content with tabs and collapsible
```

Comment:
Suggest:

    This function converts the rich text content within tabs and collapsible components to textangular format.

Reply:
Done!